ha-example/README.md

```diff
@@ -0,0 +1,198 @@
+
+When installing the Flink Kubernetes operator using `helm`, we need to add these 2 parameters into the existing default configuration string.
+This can be done via the command line using the `--set` flag:
+```
+helm install flink-kubernetes-operator flink-operator-repo/flink-kubernetes-operator \
+--set podSecurityContext=null \
+--set defaultConfiguration."log4j-operator\.properties"=monitorInterval\=30 \
+--set defaultConfiguration."log4j-console\.properties"=monitorInterval\=30 \
+--set replicas=2 \
+--set defaultConfiguration."flink-conf\.yaml"="kubernetes.operator.metrics.reporter.slf4j.factory.class\:\ org.apache.flink.metrics.slf4j.Slf4jReporterFactory
+kubernetes.operator.metrics.reporter.slf4j.interval\:\ 5\ MINUTE
+kubernetes.operator.reconcile.interval:\ 15\ s
+kubernetes.operator.observer.progress-check.interval:\ 5\ s
+kubernetes.operator.leader-election.enabled:\ true
+kubernetes.operator.leader-election.lease-name:\ flink-operator-lease" \
+-n flink
+```
+Note: The `replicas` configuration is to set to the total number of Flink Kubernetes operator replicas you want.
+This will include the leader plus the number of standby instances you want.
+One standby (two replicas in total) is usually sufficient.
+
+After running this command, you should see that there are two instances of the Flink Kubernetes operator running.
+```
+kubectl  get pod -n flink | grep flink-kubernetes-operator
+NAME                                          READY   STATUS    RESTARTS         AGE
+flink-kubernetes-operator-6cd86cc8-fmb2x      2/2     Running   0                3h
+flink-kubernetes-operator-6cd86cc8-g298v      2/2     Running   0                3h
+```
+
+And when checking the `lease` resource, we can see hte holder(leader) operator's name.
+```
+kubectl get lease -n flink
+NAME                       HOLDER                                      AGE
+flink-operator-lease       flink-kubernetes-operator-6cd86cc8-g298v    45h
+```
+
+You can test the high availability fail-over between the instances, by deleting the leader operator pod.
+You should then see that the holder will change to the other instance (or the newly created instance if the creation happens quickly).
+
+## Flink Job Manager
+
+The Job Manager ensures consistency during recovery across Task Managers.
+For the Job Manager itself to recover consistently,
+an external service must store a minimal amount of recovery metadata (like “ID of last committed checkpoint”),
+as well as information needed to elect and lock which Job Manager is the leader (to avoid split-brain situations).
+
+In order to configure Job Managers in your Flink Cluster for high availability you need to add the following settings to the configuration in your `FlinkDeplyment` CR like this:
+```yaml
+apiVersion: flink.apache.org/v1beta1
+kind: FlinkDeployment
+metadata:
+  name: recommendation-app
+spec:
+  image: quay.io/streamshub/flink-sql-runner:v0.0.1
+  flinkVersion: v1_19
```

Comment:
We should unify flink version here and in docs bellow (1.19 vs 1.20)

Reply:
Good point! Updated.